src/foam/u2/table/UnstyledTableGroup.js

```diff
@@ -75,6 +76,16 @@ foam.CLASS({
       style({ 'min-width': this.table.tableWidth_$ });
       [prop, objReturned] = this.getCellData(objForCurrentProperty, this.table.groupBy, nestedPropertiesObjsMap);
       this.start().addClass(self.table.myClass('group-content'),'h500')
+        .startContext({data: self})
+          .start(self.EXPAND)
+            .addClass(self.table.myClass('expand-icon'))
+            .callIfElse(self.table.config.allowGroupCollapse, function() {
```

Comment:
This call if else could be on line 79 then you dont need the hacked visibility hidden either